rule.go

```diff
@@ -40,3 +42,14 @@ func NewRule() *Rule {
 		Flow:              -1,
 	}
 }
+
+// RulePortRange creates rule sport/dport range.
+func NewRulePortRange(start, end uint16) *RulePortRange {
```

Comment:
Why not making this a method of `Rule` structure ?
```
(r *Rule) AddPortRange(start, end uint16)
```

Reply:
Thanks for review @aboch 
`(r *Rule) AddPortRange(start, end uint16)` may need another parameter to specify it's a Src or a Dst port range, which might add more complexity.
IMO, current fields of `Rule` structure is assigned by users manually, so `NewRulePortRange` is enough.